src/applications/disability-benefits/2346/config/form.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-sequences */
```

Comment:
Why are these rules being disabled?

Reply:
These are generated by the va forms generator by default so I don't know.

Reply:
Try removing these and see if you encounter any linter errors.

Reply:
They have been removed.

Reply:
If you've done it locally, would you commit and push those changes?

Reply:
My fault the commit has been pushed